### PR TITLE
Add pull request template via upload

### PR DESCRIPTION
**Pull request by NxFerrara**
## Changes:
- Concise description of key changes made.

## Considerations:
- Any other relevant notes or dependencies.

### CODE REVIEW

**Comment by NxFerrara** [comment]
The intended result is that GitHub will pull this template into the PR description whenever a PR is created.

File: docs/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Changes:
+- Concise description of key changes made.
+
+## Testing & Validation:
+- For feature, bug, and improvement: Describe the testing approach and how the changes were verified.
+- For test development: Explain how the new or updated tests were validated to ensure they work as intended and cover the required scenarios.
+
```
Comment (jbesoto) [inline docs/PULL_REQUEST_TEMPLATE.md:7]: Remove extra newline.

**Review by jbesoto — APPROVED** [review]
I am not sure what the purpose of this section is for or how I would even use it. Offer some clarification in the comments or make it more clear in the instructions for the section. I am not sure if this is also entirely relevant. I have been browsing the biggest open-source projects and most don't even have a template for tickets (haha). So, Occam's razor: the simplest version is probably the best version.

**Review by jbesoto — APPROVED** [review]

**Comment by NxFerrara** [comment]
I realized that you are probably right and I would compromise and say let's remove the Testing & Validation part completely. I hope the other sections are self-evident, but let me know if you are still confused about those sections.

This is actually a good practice. I want to be able to make changes on an already merged PR with a deleted branch. Firstly, I've restored the deleted branch. Secondly, I've made a commit to remove the Testing & Validation section. Should the next step be reverting the PR merge or something else? Please advise @jbesoto.

**Pull request merged**